docs/streams/index.html

```diff
@@ -248,47 +248,39 @@ <h3 style="margin-top: 5.3rem;">Hello Kafka Streams</h3>
        
            <div class="code-example__snippet b-scala">
                <pre class="brush: scala;">
-                   import java.lang.Long
-                   import java.util.Properties
-                   import java.util.concurrent.TimeUnit
+                    import java.util.Properties
```

Comment:
I think, we should not have indention here for better rendering

Reply:
I followed the convention of preceding Java examples which also have indentation, but I'll remove the indentation for the Scala example.